relay-general/src/store/transactions/processor.rs

```diff
@@ -317,11 +318,11 @@ fn scrub_identifiers_with_regex(
 
         // Sort by the capture end position.
         caps.sort_by_key(|(capture, _)| capture.end());
-        let mut changed = String::with_capacity(trans.len());
+        let mut changed = String::with_capacity(trans.len() + caps.len() * replacer.len());
```

Comment:
Not sure if this was necessary, but since the new string may now be larger than the previous one I wanted to make sure no unnecessary allocations are made.